opentelekomcloud/resource_opentelekomcloud_as_configuration_v1.go

```diff
@@ -79,18 +80,23 @@ func resourceASConfiguration() *schema.Resource {
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
 									"size": {
-										Type:     schema.TypeInt,
-										Required: true,
+										Type:         schema.TypeInt,
+										Required:     true,
+										ValidateFunc: validation.IntBetween(1, 32768),
 									},
 									"volume_type": {
-										Type:         schema.TypeString,
-										Required:     true,
-										ValidateFunc: resourceASConfigurationValidateVolumeType,
+										Type:     schema.TypeString,
+										Required: true,
+										ValidateFunc: validation.StringInSlice([]string{
+											"SATA", "SAS", "SSD", "co-p1", "uh-l1",
```

Comment:
Please use `validateVolumeType` function for this validation (as `CustomizeDiff`)

Reply:
we can't validate volume types, because we don't have AZ